pkgs/racket-doc/scribblings/reference/format.scrbl

```diff
@@ -425,7 +425,7 @@ the resulting string is appended to the significand:
 ]}
 
 ]
-@history[#:changed "8.5.0.3"
+@history[#:changed "8.5.0.5"
```

Comment:
It's actually 8.5.0.7 now... (https://github.com/racket/racket/blob/master/pkgs/base/info.rkt#L17)

Reply:
Well, close enough